content/_changelogs/12.0.0.md

```diff
@@ -1,20 +1,167 @@
+- The `cy.server()` and `cy.route()` commands have been removed. Additionally,
+  the corresponding `Cypress.Server.defaults` API has also been removed. Use the
+  [`cy.intercept()`(/api/commands/intercept) command to stub network responses
+  and requests. Addresses
+  [#22126](https://github.com/cypress-io/cypress/issues/22126).
+- The Cookie commands now uses the `hostname` as the domain by default instead
+  of the `superdomain`. This change aligns Cypress' cookie rules with the
+  browser cookie rules. This may affect what cookies are returned by
+  [`cy.getCookie()`](/api/commands/getcookie), what cookies are set with
+  [`cy.setCookie()`](/api/commands/setcookie), and the cookies cleared with
+  [`cy.clearCookies()`](/api/commands/clearcookies). Addresses
+  [#363](https://github.com/cypress-io/cypress/issues/363),
+  [#5723](https://github.com/cypress-io/cypress/issues/5723) and
+  [#24526](https://github.com/cypress-io/cypress/issues/24526)
+- Cypress now throws an error if any Cypress commands are invoked from inside a
+  [`.should()`](/api/commands/should) callback. This previously resulted in
+  unusual and undefined behavior. Addresses
+  [#5963](https://github.com/cypress-io/cypress/issues/5963),
+  [#14656](https://github.com/cypress-io/cypress/issues/14656) and
+  [#22587](https://github.com/cypress-io/cypress/issues/22587).
+
+**Features:**
+
+- Added a new configuration option called `testIsolation`, which defaults to
+  `true`. This option determines whether or not
+  [test isolation](/guides/core-concepts/writing-and-organizing-tests#Test-Isolation)
+  is enabled to ensure a clean browser context between tests. Addressed in
+  [#22230](https://github.com/cypress-io/cypress/pull/22230).
+- The previously experimental Cypress command, called
+  [`cy.origin()`](/api/commands/origin), is now generally available in
+  end-to-end testing. This command allows testing multiple domains of different
+  origins in a single test. Addresses
+  [#17336](https://github.com/cypress-io/cypress/issues/17336).
+- The previously experimental Cypress command, called
+  [`cy.session()`](/api/commands/session), is now generally available. This
+  command caches and restores `cookies`,
+  [`localStorage`](https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage),
+  and
+  [`sessionStorage`](https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage)
+  in order to recreate a consistent browser context between tests. Addresses
+  [#20977](https://github.com/cypress-io/cypress/issues/20977).
+- Added a new Cypress command, called
+  [`cy.getAllLocalStorage()`](/api/commands/getAllLocalStorage), to get
+  [`localStorage`](https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage)
+  data for all origins with which the test has interacted. Addresses
+  [#24276](https://github.com/cypress-io/cypress/issues/24276).
+- Added a new Cypress command, called
```

Comment:
This appears to be a duplicate of the above item.

Reply:
oops!